packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-table.jsx

```diff
@@ -0,0 +1,101 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { CSSTransition } from 'react-transition-group';
+import { Icon, Text, ThemedScrollbars } from '@deriv/components';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+import BarriersList from './barriers-list.jsx';
+
+const BarriersTable = ({
+    barrier_1,
+    onChange,
+    setHoveredBarrier,
+    is_barriers_table_expanded,
+    turbos_barrier_choices,
+    toggleBarriersTable,
+}) => {
+    const [chosen_barrier, setChosenBarrier] = React.useState(barrier_1);
```

Comment:
```suggestion
    const [selected_barrier, setSelectedBarrier] = React.useState(barrier_1);
```
@kate-deriv just a nit-pick😊

Reply:
Thank you, I'll rename it